homeassistant/components/raspihats.py

```diff
@@ -0,0 +1,251 @@
+"""
+Support for controlling raspihats boards.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/raspihats/
+"""
+import logging
+import threading
+import time
+
+from homeassistant.const import (
+    EVENT_HOMEASSISTANT_START, EVENT_HOMEASSISTANT_STOP
+)
+
+REQUIREMENTS = ['raspihats==2.2.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'raspihats'
+
+CONF_I2C_HATS = 'i2c_hats'
+CONF_BOARD = 'board'
```

Comment:
All the unused variables will be used by 2 new platforms switch.raspihats and binary_sensor.raspihats that I have already prepared. I will make PRs for those 2 new platforms after this one is accepted.